Dockerfile

```diff
@@ -1,37 +1,31 @@
 FROM debian:jessie
 
-ENV START_MAP ${START_MAP}
-ENV SERVER_NAME ${SERVER_NAME}
-ENV MAXPLAYERS ${MAXPLAYERS}
-ENV START_MONEY ${START_MONEY}
-ENV BUY_TIME ${BUY_TIME}
-ENV FRIENDLY_FIRE ${FRIENDLY_FIRE}
-ENV MAP_AUTOKICK ${MAP_AUTOKICK}
-ENV TIMELIMIT ${TIMELIMIT}
-ENV MAP_MAXROUNDS ${MAP_MAXROUNDS}
-ENV ROUNDTIME ${ROUNDTIME}
-ENV WINLIMIT ${WINLIMIT}
-ENV MAP_VOTE_RATIO ${MAP_VOTE_RATIO}
-ENV MAP_FREEZETIME ${MAP_FREEZETIME}
-ENV ADMIN_STEAM ${ADMIN_STEAM}
-ENV MAP_AUTOTEAMBALANCE ${MAP_AUTOTEAMBALANCE}
-ENV USE_VOTEMAP ${USE_VOTEMAP}
-ENV RCON_PASSWORD ${RCON_PASSWORD}
-ENV SERVER_PASSWORD ${SERVER_PASSWORD}
-
-RUN apt update && apt install lib32gcc1 -y
+RUN apt-get update && apt-get -y --force-yes install lib32gcc1 wget zip

```

Comment:
```suggestion
RUN apt-get update && apt-get -y --force-yes install lib32gcc-s1 wget zip

```

Reply:
well, debian made changes in the packages, so yes, it could be broken today, thx for the fix)